Traceback (most recent call last):
  File "/home/ubuntu/fccre/app.py", line 1, in <module>
    from flask import Flask, render_template, request, redirect, url_for, flash, send_file
ModuleNotFoundError: No module named 'flask'
 * Serving Flask app 'app'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 884-194-987
127.0.0.1 - - [22/Jun/2025 10:21:20] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:21:20] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:21:53] "GET /contact/167 HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:21:53] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:22:19] "[32mPOST /contact/167 HTTP/1.1[0m" 302 -
127.0.0.1 - - [22/Jun/2025 10:22:19] "GET /contact/167 HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:22:20] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
 * Serving Flask app 'app'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [22/Jun/2025 10:27:04] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:27:04] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:27:07] "GET /contact/167 HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:27:07] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:27:15] "GET /contact/167 HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:27:15] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:27:22] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:27:22] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:27:35] "GET /contact/171 HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:27:36] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:30:39] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:30:39] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:30:40] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:31:26] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:31:26] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [22/Jun/2025 10:31:31] "GET /contact/167 HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:31:31] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
 * Detected change in '/home/ubuntu/fccre/app.py', reloading
DEBUG: Redirection vers modifier_contact avec id=167
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 884-194-987
127.0.0.1 - - [22/Jun/2025 10:33:32] "GET /contact/167 HTTP/1.1" 200 -
127.0.0.1 - - [22/Jun/2025 10:33:32] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
